src/magicbox/SuggestionsManager.ts

```diff
@@ -267,38 +305,35 @@ export class SuggestionsManager {
   }
 
   private buildPreviewContainer() {
-    return $$('div', {
-      className: 'coveo-preview-container'
-    }).el;
-  }
-
-  private get querySuggestPreviewComponent() {
-    const querySuggestPreviewElement: HTMLElement = $$(this.root).find(`.${Component.computeCssClassNameForType('QuerySuggestPreview')}`);
-    if (!querySuggestPreviewElement) {
-      return;
-    }
-    return Component.get(querySuggestPreviewElement);
+    return (this.previewContainer = $$(
+      'div',
+      {
+        className: 'coveo-preview-container'
+      },
+      $$('h4', {
```

Comment:
Is `h4` the semantically correct tag here ? I'd be surprised it works in all scenario.

I guess we'd be better with a simple div.

Reply:
I doubt it is, I copied it from `QuerySuggestPreview`. I'll change it to a `div`.